src/kernel/log.zig

```diff
@@ -0,0 +1,37 @@
+const serial = @import("serial.zig");
+const fmt = @import("std").fmt;
+
+pub const Level = enum {
+    INFO,
+    DEBUG,
+    WARNING,
+    ERROR
+};
+
+pub fn init() void {
+    serial.init(serial.DEFAULT_BAUDRATE, serial.Port.COM1) catch unreachable;
+}
+
+fn logCallback(context: void, str: []const u8) anyerror!void {
+    for (str) |char| {
```

Comment:
U have a writeStr in ur serial interface

Reply:
Indeed. This is from before that was added.